core/metadata.js

```diff
@@ -248,11 +251,11 @@ export type PlatformIncompatibility = PathIssue & {docType: string}
 // TODO: return null instead of an empty array when no issue was found?
 function detectPlatformIncompatibilities (metadata /*: Metadata */, syncPath /*: string */) /*: Array<PlatformIncompatibility> */ {
   const {path, docType} = metadata
-  const pathLenghIssue = detectPathLengthIssue(join(syncPath, path), platform)
+  const pathLenghIssue = detectPathLengthIssue(nodepath.join(syncPath, path), platform)
```

Comment:
What about dropping the `const {path, docType} = metadata` statement above and use `metadata.path` and `metadata.docType` explicitely? Wouldn't the change be smaller and not requiring a different `nodepath` module alias from other places?

Reply:
Why not